### DISCUSSION TITLE
Give account to friend

### DISCUSSION THREAD

**Discussion by jlivingstonsg** [community/community#22421] [New to GitHub]
<p>Hi.<br>
We did a repo together , some friends,<br>
and now we decided to give “one  Contributor account” to another  “Contributor account”.</p>
<p>So he added another Contributor email account to try to give the account to another Contributor .</p>
<p>It failed.</p>
<p>How do I give my account to another?</p>
<p>Regards</p>

**Comment by jlivingstonsg** [comment, marked as answer]
<p>Found this.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/3172436/181070842-6d270932-ebf6-48ef-b229-d23b6b2e2b1a.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository" target="_blank" rel="noopener nofollow ugc">Transferring a repository - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Sorry for posting</p>

**Comment by mpboom** [comment]
<p>Hey <a class="mention" href="/jlivingstonsg">@jlivingstonsg</a> - don’t be sorry for posting! Thank you so much for sharing your solution with the Community after finding it. That helps other people facing the same issue, too :slight_smile:</p>
